pages/TheLearnerView.html

```diff
@@ -20,7 +20,7 @@ <h2>
     sequence.
   </p>
 
-  <img src="static/images/exploration.png" width="620">
+  <img src="static/images/exploration.png" width="620" alt="Two white cards adjacent to each other, one containing a blue box asking the learner to create a single note as part of the music theory exploration. The second card consists of a white 'play' button and a single musical note illustrated.">
```

Comment:
See above for a suggested rewording.

Reply:
Two white cards adjacent to each other. The first card contains instructions to create a musical note. The second card has a single musical note illustrated, along with a play button.

Does this work better?

Reply:
Yes!